MeesGame/Gameplay/GameObjects/TileField.cs

```diff
@@ -10,8 +10,19 @@ public class TileField : GameObjectGrid, IDiscreteField, ITileField
         private static readonly Point NO_POINT = new Point(-1, -1);
         private static readonly Point DEFAULT_START = new Point(1, 1);
         private Point start = NO_POINT;
-        private int sightRadius = 3;
 
+        public Point Start
+        {
+            get
+            {
```

Comment:
I think something went wrong while merging the master branch because this property was relocated